### PR TITLE
change UI parameters and tooltips

### PR DESCRIPTION
**Pull request by siyaramblackduck**

### CODE REVIEW

**Review by sadmananik — CHANGES_REQUESTED** [review]

File: blackduck-security-task/task.json [hunk 1]
```diff
@@ -994,14 +994,14 @@
...
       "groupName": "blackduckScaToolOptionsForSrm"
     },
     {
       "name": "detectExecutionPath",
       "type": "string",
-      "label": "Detect Installation Path",
+      "label": "Detect Execution Path",
       "defaultValue": "",
       "required": false,
       "helpMarkDown": "Path to the Black Duck SCA Detect jar file to use.",
```
[the remark this answers is not in the document]
Comment (siyaramblackduck) [reply blackduck-security-task/task.json]: done

File: blackduck-security-task/task.json [hunk 2]
```diff
@@ -705,7 +705,7 @@
     {
       "name": "coverityExecutionPath",
       "type": "string",
-      "label": "Coverity Installation Path",
+      "label": "Coverity Execution Path",
       "defaultValue": "",
       "required": false,
       "helpMarkDown": "Path to Coverity CLI.",
```
[the remark this answers is not in the document]
Comment (siyaramblackduck) [reply blackduck-security-task/task.json]: done

File: blackduck-security-task/task.json [hunk 3]
```diff
@@ -931,10 +931,10 @@
     {
       "name": "detectInstallDirectory",
       "type": "string",
-      "label": "Detect Installation Path",
+      "label": "Detect Installation Directory",
       "defaultValue": "",
       "required": false,
-      "helpMarkDown": "Directory path to install Detect.",
+      "helpMarkDown": "Provide the Detect installation path on the agent where it will be installed during the build",
```
[the remark this answers is not in the document]
Comment (siyaramblackduck) [reply blackduck-security-task/task.json]: done

File: blackduck-security-task/task.json [hunk 4]
```diff
@@ -615,10 +615,10 @@
     {
       "name": "coverityInstallDirectory",
       "type": "string",
-      "label": "Installation Path",
+      "label": "Installation Directory",
       "defaultValue": "",
       "required": false,
-      "helpMarkDown": "Directory path to install Coverity.",
+      "helpMarkDown": "Provide the Coverity installation path on the agent where it will be installed during the build.",
```
[the remark this answers is not in the document]
Comment (siyaramblackduck) [reply blackduck-security-task/task.json]: done

**Review by sadmananik — APPROVED** [review]

**Pull request merged**